manager/base.py

```diff
@@ -665,15 +667,35 @@ def parse_tags(tags_list):
     return tags
 
 
+def inventory_groups_condition(group_ids) -> Tuple[str, bool]:
+    """Prepares SQL condition (array of possible values) for filtering by inventory group IDs"""
+    res = []
+    include_ungrouped = False
+    for elem in group_ids:
+        if elem == []:
+            include_ungrouped = True
+        else:
+            res.append("\"" + json.dumps(elem).replace("\"", "\\\"") + "\"")
+    return "{" + ",".join(res) + "}", include_ungrouped
```

Comment:
you might be able to handle escapes by passing it to another `json.dumps`, I haven't tried it tho, it looks way better now anyways

Reply:
Yea it works, thanks